etl/steps/data/meadow/ggdc/2022-11-28/penn_world_table.meta.yml

```diff
@@ -0,0 +1,153 @@
+dataset:
+  sources:
+    -
+      name: Penn World Table 10.0
+      published_by: Feenstra, R. C., Inklaar, R. and Timmer, M.P. (2015), "The Next Generation of the Penn World Table". American Economic Review, 105(10), 3150-3182
+
+tables:
+  penn_world_table:
+    variables:
+      rgdpe:
```

Comment:
General advice: It's better to use descriptive variable names that are easy to understand without having to look up its meaning in a table. But I understand that for this dataset it's a bit tricky because there are many, with very specific meanings.